C++/README.md

```diff
@@ -12,6 +12,7 @@ Implementation of well-known (and some rare) algorithms, in C++.
 * Dynamic programming
   * [Matrix chain multiplication](source/dynamic_programming/matrix_chain_mult.cpp) :white_check_mark:
   * Maximum sum contiguous subarray: [Kadane's algorithm](source/maximum_subarray/kadane.cpp)
+  * [Longest_Subsequence](include/dynamic_programming/longest_subsequence.hpp) :white_check_mark:
```

Comment:
"Longest increasing or decreasing subsequence" rather than "Longest_Subsequence". Faheel might have a different opinion on whether to include the increasing/decreasing part, but definitely no underscore.